src/client/src/containers/AzureSubscriptions/index.tsx

```diff
@@ -110,29 +119,81 @@ class AzureSubscriptions extends React.Component<Props, IState> {
     }
     return () => {};
   }
+
+  public getServicesOrganizer(
+    type: string | undefined,
+    isLoggedIn: boolean,
+    setDetailPage: any,
+    title: any
+  ) {
+    const { formatMessage } = this.props.intl;
+    return (
+      <div
+        className={classnames(styles.servicesContainer, {
+          [styles.overlay]: !isLoggedIn
+        })}
+      >
+        <div className={styles.servicesCategory}>
+          <div className={styles.categoryTitle}>{type}</div>
+          <div className={styles.categoryDescriptor}>
+            {formatMessage(title)}
+          </div>
+          <div className={styles.servicesCategoryContainer}>
+            {azureServiceOptions.map(option => {
+              if (option.type === type) {
+                return (
+                  <div
+                    key={JSON.stringify(option.title)}
+                    className={classnames(styles.subscriptionCardContainer)}
+                  >
+                    <Card
+                      option={option}
+                      buttonText={this.addOrEditResourceText(
+                        option.internalName
+                      )}
+                      handleButtonClick={this.getServicesModalOpener(
+                        option.internalName
+                      )}
+                      disabled={!isLoggedIn}
+                      handleDetailsClick={setDetailPage}
+                      useNormalButtons={this.isSelectionCreated(
+                        option.internalName
+                      )}
+                    />
+                  </div>
+                );
+              }
+            })}
+          </div>
+        </div>
+      </div>
+    );
+  }
   public render() {
     const { isLoggedIn, setDetailPage } = this.props;
+    const serviceTypes = azureServiceOptions.map(option => option.type);
+    const uniqueServiceTypes = serviceTypes.filter(
+      (type, index, types) => types.indexOf(type) === index
+    );
     return (
       <div className={styles.container}>
-        {azureServiceOptions.map(option => (
-          <div
-            key={JSON.stringify(option.title)}
-            className={classnames(styles.subscriptionCardContainer, {
-              [styles.overlay]: !isLoggedIn
-            })}
-          >
-            <Card
-              option={option}
-              buttonText={this.addOrEditResourceText(option.internalName)}
-              handleButtonClick={this.getServicesModalOpener(
-                option.internalName
-              )}
-              disabled={!isLoggedIn}
-              handleDetailsClick={setDetailPage}
-              useNormalButtons={this.isSelectionCreated(option.internalName)}
-            />
-          </div>
-        ))}
+        {uniqueServiceTypes.map(serviceType => {
+          let categoryTitle;
+          switch (serviceType) {
```

Comment:
If you change the `servicesEnum` you could simply do here `messages[serviceType + 'Title']`

Reply:
will dismiss this for now as approaching it this way does not seem to work with our application